content/blog/2021/machine-learning.md

```diff
@@ -0,0 +1,63 @@
+Title: Machine learning: A major cause of the Pythonic buzz
+Date: 2021-08-24 12:05
+Author: Anuran Roy (@anuran-roy)
+Category: 2021
+Slug: machine-learning-pycon
+Summary: What is machine learning? Where does Python play its part? What's in store for you at PyCon India 2021 for Machine Learning?
+
+## Machine learning: A major cause of the Pythonic buzz
+
+
+<p class="text-center">
+  <img src="/content/images/ml-article-2021-Illustration.png" alt="Python libraries for AI and data"/>
+</p>
+
+ **“Predicting the future isn’t magic, it’s artificial intelligence.” ~Dave Waters**
+
+<p class="text-center">
+  <img src="/content/images/ml-article-2021-Relations.png" alt="Relations between AI, ML, DL, Big Data and Data Science"/>
+</p>
+
```

Comment:
```suggestion
 **“Predicting the future isn’t magic, it’s artificial intelligence.” ~Dave Waters**

<p class="text-center">
  <img src="/content/images/ml-article-2021-Relations.png" alt="Relations between AI, ML, DL, Big Data and Data Science"/>
</p>
```

I think only one image would suffice

Reply:
Also, I changed it from HTML to markdown syntax.

Reply:
Done.